components/bl-line-chart-component/component.json

```diff
@@ -0,0 +1,109 @@
+{
+  "id": "c_c4f766d585f372396136476db06960af",
+  "name": "Line Chart",
+  "description": "The Line Chart component allows you to display data in the form of charts. You can also choose a chart type.",
+  "showInToolbox": true,
+  "faIcon": "chart-line",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "checkbox",
+      "name": "disabled",
+      "label": "Disabled",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "disabledLogic",
+      "handlerLabel": "Disabled Logic",
+      "dataBinding": true,
+      "handlerDescription": "This handler allows you to disable a component."
+    },
+    {
+      "type": "text",
+      "name": "height",
+      "label": "Height",
+      "showInSettings": true,
+      "defaultValue": "400px",
+      "hasLogicHandler": true,
+      "handlerId": "heightLogic",
+      "handlerLabel": "Height Logic",
+      "handlerDescription": "This handler allows you to specify the height of a component.",
+      "dataBinding": true
+    },
+    {
+      "type": "text",
+      "name": "width",
+      "label": "Width",
+      "showInSettings": true,
+      "defaultValue": "500px",
+      "hasLogicHandler": true,
+      "handlerId": "widthLogic",
+      "handlerLabel": "Width Logic",
+      "handlerDescription": "This handler allows you to specify the width of a component.",
+      "dataBinding": true
+    },
+    {
+      "type": "select",
+      "name": "type",
+      "label": "Type",
+      "showInSettings": true,
+      "defaultValue": "line",
+      "hasLogicHandler": true,
+      "handlerId": "typeLogic",
+      "handlerLabel": "Type Logic",
+      "handlerDescription": "This handler allows you to specify the chart type.",
+      "dataBinding": true,
+      "options": [
+        {
+          "value": "bar",
+          "label": "Bar"
+        },
+        {
+          "value": "line",
+          "label": "Line"
+        },
+        {
+          "value": "pie",
+          "label": "Pie"
+        }
+      ]
+    },
+    {
+      "type": "json",
+      "name": "labels",
+      "label": "Labels",
+      "showInSettings": true,
+      "defaultValue": "[\"A\", \"B\", \"C\", \"D\"]",
```

Comment:
I would rather see some "real" data as default values. For example day names as Labels, and Sold Apples and Piers at legend. But not A,B,C,D and test1,test2 as I see at preview.

Reply:
Also, if it possible we should sync default values here and at screenshots with examples.